src/org/daisy/dotify/api/formatter/MarkerIndicator.java

```diff
@@ -6,17 +6,19 @@
  * @author Joel Håkansson
  */
 public class MarkerIndicator {
-    private final String name, indicator;
+    private final String name, indicator, textStyle;
 
     /**
      * Creates a new marker indicator with the specified parameters.
      *
      * @param name      the name of the markers to indicate
      * @param indicator the string indicating an occurrence
+     * @param textStyle the text style
      */
-    public MarkerIndicator(String name, String indicator) {
+    public MarkerIndicator(String name, String indicator, String textStyle) {
         this.name = name;
         this.indicator = indicator;
+		this.textStyle = textStyle;
```

Comment:
The textStyle is never used, code does not compile. Missing getter function.

Reply:
OK. I probably forgot to commit something.

Reply:
Hope I've fixed it now.

Reply:
Ran check 
```
File contains tab characters (this is the first instance).  - line 21
```